### DISCUSSION TITLE
Very Long Time-Lapse Functions

### DISCUSSION THREAD

**Discussion by PMyers21** [gopro/labs#671] [General]
Is there a way to do intervals longer than 1/photo every 10 mins? 

I'm trying to set a GP up for a construction project and would like longer interval options.

Thanks

**Comment by dnewman-gpsw** [comment]
All recent cameras (without Labs) have a photo every hour option.
![GRAB-20231130-192721a](https://github.com/gopro/labs/assets/10856084/8b12a562-2d6c-4aa7-915a-7be62853f501)

With Labs you have much more control like: https://gopro.github.io/labs/control/construction/
